src/Screen.php

```diff
@@ -3,18 +3,54 @@
 namespace Native\Laravel;
 
 use Native\Laravel\Client\Client;
+use Illuminate\Support\Facades\Log;
 
 class Screen
 {
-    public function __construct(protected Client $client) {}
+  public function __construct(protected Client $client)
+  {
+  }
 
-    public function cursorPosition(): object
-    {
-        return (object) $this->client->get('screen/cursor-position')->json();
-    }
+  public function getClient()
+  {
+    return $this->client;
+  }
```

Comment:
Where is this used?

Reply:
sorry made that for debug and forgot to clean after myself